### PR TITLE
update jsut recipe

### PR DESCRIPTION
**Pull request by DrWelles**

### CODE REVIEW

File: egs/jsut/svs1/conf/tuning/train_naive_rnn.yaml.tmp [hunk 1]
```diff
@@ -0,0 +1,85 @@
+
```
Comment (ftshijt) [inline egs/jsut/svs1/conf/tuning/train_naive_rnn.yaml.tmp:1]: this is .tmp

File: egs/jsut/svs1/run.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+train_config=conf/train.yaml
+inference_config=conf/decode.yaml
+
+# text related processing arguments
+g2p=none
+cleaner=none
+
+./svs.sh \
+    --lang jp \
+    --stage 5 \
```
Comment (ftshijt) [inline egs/jsut/svs1/run.sh:40]: remove this line

File: egs/svs1/cmd.sh [hunk 3]
```diff
@@ -0,0 +1,110 @@
+# ====== About run.pl, queue.pl, slurm.pl, and ssh.pl ======
```
Comment (ftshijt) [inline egs/svs1/cmd.sh:1]: what is this?

File: egs/svs1/conf/decode.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# This configuration is the decoding setting for FastSpeech or FastSpeech2.
```
Comment (ftshijt) [inline egs/svs1/conf/decode.yaml:1]: You are having egs/svs1, please remove them

**Review by ftshijt — COMMENTED** [review]
You also add the midi_dump, please remove them

File: egs/svs1/cmd.sh (hunk 3 above)
Comment (DrWelles) [reply egs/svs1/cmd.sh:1]: > what is this?

I don't know. I just copy this from nit.

Comment (ftshijt) [reply egs/svs1/cmd.sh:1]: I‘m referring that you are having egs/svs1, which is not related to a specific recipe

Comment (DrWelles) [reply egs/svs1/cmd.sh]: > I‘m referring that you are having egs/svs1, which is not related to a specific recipe

okay,  sorry for this

File: egs/jsut/svs1/conf/tuning/train_xiaoice_noDP.yaml [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+best_model_criterion:
+-   - valid
+    - loss
+    - min
+-   - train
+    - loss
+    - min
+
+vocoder_checkpoint: /home/exx/jiatong/projects/svs/ParallelWaveGAN/egs/opencpop/voc1/exp/tr_no_dev_opencpop_hifigan.v1/checkpoint-250000steps.pkl
+gpu_id: 0
```
Comment (ftshijt) [inline egs/jsut/svs1/conf/tuning/train_xiaoice_noDP.yaml:92]:
```suggestion
```

**Review by ftshijt — COMMENTED** [review]
You seem also accidentally include midi_dump into the PR, please remove it (you may also update gitignore for `midi_dump` and `wav_dump`

**Comment by ftshijt** [comment]
Please also add a pre-trained model in a later PR.

**Pull request merged**